web/src/redux/actions/editApd/previousActivities.js

```diff
@@ -41,91 +41,37 @@ export const setPreviousActivityFederalActualExpenseForHITandHIE = (
 });
 
 /**
- * Set the total amount approved for MMIS activities at the 50/50 match level
- * in a previous federal fiscal year.
+ * Set the total amount approved for MMIS activities for MMIS APDs
  * @param {String} year Federal fiscal year, four-digit
  * @param {Number} expense Total approved HIE/HIT for the year
+ * @param {Number} level FFP Percentage, [50, 75, 90]
+ * @param {String} fundingType DDI or M&O
  */
-export const setPreviousActivityApprovedExpenseforMMIS50FFP = (
+export const setPreviousActivityApprovedExpense = (
   year,
-  expense
-) => ({
-  type: EDIT_APD,
-  path: `/previousActivities/actualExpenditures/${year}/mmis/50/totalApproved`,
-  value: expense
-});
-
-/**
- * Set the total amount approved for MMIS activities at the 75/25 match level
- * in a previous federal fiscal year.
- * @param {String} year Federal fiscal year, four-digit
- * @param {Number} expense Total approved HIE/HIT for the year
- */
-export const setPreviousActivityApprovedExpenseforMMIS75FFP = (
-  year,
-  expense
-) => ({
-  type: EDIT_APD,
-  path: `/previousActivities/actualExpenditures/${year}/mmis/75/totalApproved`,
-  value: expense
-});
-
-/**
- * Set the total amount approved for MMIS activities at the 90/10 match level
- * in a previous federal fiscal year.
- * @param {String} year Federal fiscal year, four-digit
- * @param {Number} expense Total approved HIE/HIT for the year
- */
-export const setPreviousActivityApprovedExpenseforMMIS90FFP = (
-  year,
-  expense
+  expense,
+  level,
+  fundingType
```

Comment:
It looks like we've been referring to "DDI" and "M&O" as a `fundingCategory`. I wonder if we can make that consistent?